byok/Containerfile.tool

```diff
@@ -1,20 +1,27 @@
 ARG BYOK_TOOL_IMAGE=quay.io/$USERNAME/tool:latest
 ARG UBI_BASE_IMAGE=registry.access.redhat.com/ubi9/ubi:latest
+ARG HERMETIC=false
 FROM ${UBI_BASE_IMAGE}
 ARG LOG_LEVEL=info
 ARG OUT_IMAGE_TAG=byok-image
 ARG BYOK_TOOL_IMAGE
 ARG UBI_BASE_IMAGE
+ARG HERMETIC
 RUN dnf install -y buildah python3.11 python3.11-pip
 
 USER 0
 WORKDIR /workdir
 
-COPY requirements.cpu.txt .
-RUN pip3.11 install --no-cache-dir -r requirements.cpu.txt
+COPY requirements.gpu.txt .
```

Comment:
@raptorsun Hi, suppose the `quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/lightspeed-rag-tool:{{revision}}` is built based on it which is requirements.gpu.txt, could we use this tool image on VM which has no GPU? thanks.

Reply:
yes, this works on CPU only machines, too. 
the disavantage is the PyTorch package is much larger than the CPU only build.

Reply:
Thanks!